polyglot/piranha/demo/match_only/go/configurations/rules.toml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+#
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+[[rules]]
+name = "find_go_stmt_for_loop"
+query = """
```

Comment:
Can we use a constraint here? 

like query =  `(go_statement) @go_stmt`

[[rules.constraints]]
matcher = "(for_statement) @fs"
queries=[]

Reply:
it will make the rule easier and we dont have to use quantifiers (Which we know dont always work well)

Reply:
Great suggestion. Thanks. I didn't think of an empty query.